learnAndCodeFinalProject/src/main/java/org/learnAndCode/NotificationQueue.java

```diff
@@ -0,0 +1,20 @@
+package org.learnAndCode;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class NotificationQueue {
+    private static List<String> notifications = new LinkedList<>();
+
+    public static void addNotification(String message) {
+        notifications.add(message);
+    }
+
+    public static String getNextNotification() {
+        return notifications.isEmpty() ? null : notifications.remove(0);
```

Comment:
if you are passing null if no notification in the list, then you should handle this null check at the function call

Reply:
Also below function hasNotifications() could have been used here instead check for the Empty again.